typing/typeopt.ml

```diff
@@ -59,13 +63,15 @@ let maybe_pointer_type env ty =
 let maybe_pointer exp = maybe_pointer_type exp.exp_env exp.exp_type
 
 type classification =
-  | Int
+  | Int   (* any immediate type *)
   | Float
   | Lazy
   | Addr  (* anything except a float or a lazy *)
   | Any
 
-let classify env ty =
+(* Classify a ty into a [classification]. Looks through synonyms, using [scrape_ty].
+   Returning [Any] is safe, though may skip some optimizations. *)
```

Comment:
Looks like this comment about `Any` got lost, and is probably worth keeping around in some form?

Reply:
That comment about `Any` is new here. Maybe you somehow diff'd backwards?